.github/workflows/ci.yml

```diff
@@ -12,6 +15,7 @@ jobs:
   check:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
this is a drive-by, so that if the Linux build fails _for linux reasons,_ it doesn't blow up the Windows build too.